hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/VirtualBlobValueSerializer.java

```diff
@@ -55,6 +58,11 @@ public int getSerializedSize() {
         return VARIABLE_DATA_SIZE;
     }
 
+    @Override
+    public int getSerializedSize(VirtualBlobValue data) {
+        return data.getSizeInBytes();
+    }
+
     // FUTURE WORK: mark it as @Override after migration to platform 0.39
```

Comment:
It's not in the change set but could you please address this minor comment as well? Apparently, we've already migrated to 0.39

Reply:
Good catch. Will do